iaso/models/entity.py

```diff
@@ -149,7 +150,7 @@ class Entity(SoftDeletableModel):
     submission
     """
 
-    name = models.CharField(max_length=255)
+    name = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
Not blocking but just to let you know: the Django convention is to avoid using `null=True` with string-based fields (https://docs.djangoproject.com/en/4.2/ref/models/fields/#null).